packages/lab/src/overlay/OverlayPanelBase.tsx

```diff
@@ -36,16 +36,13 @@ export const OverlayPanelBase = (props: ComponentPropsWithoutRef<"div">) => {
         </Button>
         <div className={withBaseName("content")}>{children}</div>
       </div>
-
       <FloatingArrow
         {...arrowProps}
         strokeWidth={1}
         fill="var(--overlay-background)"
         stroke="var(--overlay-borderColor)"
-        style={{
-          height: "calc(var(--salt-size-adornment) + 6px)", // FIXME: +6px to account for Floating UI's FloatingArrow positioning calculation
-          width: "calc(var(--salt-size-adornment) + 8px)", // FIXME: +8px to account for Floating UI's FloatingArrow positioning calculation
-        }}
+        height={5}
```

Comment:
is there no token that represents these values? how did they come about?

Reply:
i copied them from tooltip